### ISSUE TITLE
Can this work for Godot 4+ games with multiplayer and open world chunks?

### ISSUE THREAD

**Issue by WithinAmnesia** [alejandrociatti/platinum#1]
Test chunks link: https://github.com/WithinAmnesia/ARPG/discussions/15

I'm trying to find a way to seamless load and unload chunks for a 2D multiplayer game project to make an open world with a working server using Godot 4.2.1.NET.

How can this work for multiplayer and what is needed for this to potentially work? What options can be used for chunk loading and unloading seamlessly in Godot 4.2.1.NET? Please give feedback.

**Comment by alejandrociatti** [comment]
Hey @WithinAmnesia , this game is a draft work-in-progress, with very little functionality. 
First of all, it is using an older version of Godot, then, I think I left it before adding player-to-player collision & attacks, and also without syncing the mob health across clients. It should run and work though, once you get all the servers running. 

I would say: I am far from talking about chunk-loading, and if you are looking at this repo, you probably should not even worry about it as well. 

Why don't you focus on creating a single dungeon or map first?  
Then you could spawn players in different copies of the same map once one gets too full. 

I based a lot of the code on some video tutorials, here you go, maybe this guy has more info: 

[Godot Conf 2021 ( Authoritative Multiplayer with Godot - Stefan_Gamedev )](https://www.youtube.com/watch?v=2n8mmqs-6Ps )
Stefan's channel: https://www.youtube.com/c/GameDevelopmentCenter/join 

Hope this helps! Start building simple!

**Comment by WithinAmnesia** [comment]
How does one make entities and storage systems like chests and banks for M.M.O.'s for Godot 4.2.1.NET+? 
I found a really decent base for a multiplayer seamless chunk system in the Godot_Voxel multiplayer demonstration 'Blockygame' https://github.com/WithinAmnesia/ARPG/discussions/16 + https://github.com/Zylann/voxelgame/issues/104 + https://github.com/Zylann/voxelgame/issues/103

I found a tutorial I am working my way through for a working demo and to get better at Godot coding as well: https://www.youtube.com/watch?v=V79YabQZC1s

Although I'm not sure what to do to try and get the blockygame multiplayer code to work with other examples for there is a demanding / complicated chunk loading and unloading going on with the blockygame demo where everything has to be able to load and unload and communicate with the server. What are your thoughts and suggestions? All feedback is welcome.
